crates/rslint_lexer/src/lib.rs

```diff
@@ -757,6 +757,32 @@ impl<'src> Lexer<'src> {
         }
     }
 
+    #[inline]
+    fn read_shebang(&mut self) -> LexerReturn {
+        assert_eq!(self.cur, 0);
```

Comment:
```suggestion
        debug_assert_eq!(self.cur, 0);
```
This should effectively never trigger since the logic of when the method is run is very simple